### PR TITLE
Feature/sockshop register delete user

### LINKED ISSUE #89: Action - SockShop register user

### LINKED ISSUE #90: Action - SockShop delete user
Possible demo scenario

User is generating a suspicious traffic such as creating orders and deleting them right away.
A rule could be introduced that would delete the user thus eliminating a source of a potential infrastructure attack.

### PR DESCRIPTION
**Pull request by proman3419**
Closes #89 
Closes #90

### CODE REVIEW

**Review by pgorgolew — APPROVED** [review]

File: executor/src/test/java/org/k8loud/executor/actions/cnapp/sockshop/http/deleteUser.http [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# register user
```
Comment (pgorgolew) [inline executor/src/test/java/org/k8loud/executor/actions/cnapp/sockshop/http/deleteUser.http:1]: delete not register

**Review by pgorgolew — CHANGES_REQUESTED** [review]

File: executor/src/test/java/org/k8loud/executor/actions/cnapp/sockshop/http/deleteUser.http (hunk 1 above)
Comment (proman3419) [reply executor/src/test/java/org/k8loud/executor/actions/cnapp/sockshop/http/deleteUser.http:1]: copy paste goes brrr

**Comment by proman3419** [comment]
**If there's time left**
Add check if user exists when registering, atm 500 is returned when attempting to register an existing user

**Review by pgorgolew — DISMISSED** [review]

File: executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+@ConfigurationProperties(prefix = "sockshop")
+public class SockShopProperties {
+    private String registerUserUrlSupplement = "register";
+    private String customersUrlSupplement = "customers";
```
Comment (Agwen12) [inline executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java:12]: why no static?

File: executor/src/main/java/org/k8loud/executor/service/ExecutionServiceImpl.java [hunk 3]
```diff
@@ -8,7 +8,8 @@
 
 @Slf4j
 @Service
-public class ExecutionServiceImpl implements ExecutionService {
+public class
+ExecutionServiceImpl implements ExecutionService {
```
Comment (Agwen12) [inline executor/src/main/java/org/k8loud/executor/service/ExecutionServiceImpl.java:12]: ???

File: executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java (hunk 2 above)
Comment (proman3419) [reply executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java:12]: None of our Properties classes have static fields

File: executor/src/main/java/org/k8loud/executor/service/ExecutionServiceImpl.java (hunk 3 above)
Comment (proman3419) [reply executor/src/main/java/org/k8loud/executor/service/ExecutionServiceImpl.java:12]: whoops

File: executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java (hunk 2 above)
Comment (Agwen12) [reply executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java:12]: but those variables could be just static fields in a class IMO no need for Config bean

Comment (proman3419) [reply executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java:12]: Resolved in a private conversation

**Review by pgorgolew — APPROVED** [review]

**Pull request merged**